spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/IpcTagKey.java

```diff
@@ -32,17 +32,31 @@ public enum IpcTagKey {
    */
   result("ipc.result"),
 
+  /**
+   * Indicates where the result was ultimately sourced from such as cache, direct,
+   * proxy, fallback, etc.
+   */
+  source("ipc.source"),
+
   /**
    * Dimension indicating a high level status for the request. These values are the same
    * for all implementations to make it easier to query across services. See {@link IpcStatus}
    * for permitted values.
    */
   status("ipc.status"),
 
+  /**
+   * Dimension indicating the transport-specific code that aligns to the IPC status.
+   * The values for this are implementation specific. For example with HTTP,
+   * the status code value would be used here.
+   */
+  statusCode("ipc.status.code"),
```

Comment:
Why doesn't `ipc.status.detail` serve this purpose? I don't think we need yet another status dimension.